programs/ssl/ssl_client2.c

```diff
@@ -216,10 +215,19 @@ int main( void )
 #endif /* MBEDTLS_SSL_DTLS_CONNECTION_ID */
 
 #if defined(MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED)
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL) 
+#define USAGE_PSK_RAW                                               \
+    "    psk=%%s                  default: \"\" (disabled)\n"     \
```

Comment:
Minor: The old and new branches share the documentation for `psk` and `psk_identity`. Remove this duplication?

Reply:
In 1.3 we also have to define the key_exchange_modes along with it.

Reply:
Yep, but the duplication of `psk` and `psk_identity` can be avoided, or not?